src/components/organizations/detail.js

```diff
@@ -31,7 +32,30 @@ class OrganizationDetail extends React.Component {
     this.props.dispatch(push('/organizations/' + this.props.organization.id + '/clusters/' + cluster));
   }
 
+  // determine whether the component should deal with BYOC credentials
+  // (not relevant on KVM)
+  canCredentials = (provider) => {
+    if (provider === 'aws' || provider === 'azure') {
```

Comment:
nit
`canCredentials ` -> `hasCredentials ` ?

Reply:
This function should return whether the current context (installation) supports credentials.